### ISSUE TITLE
403 Forbidden when downloading Node 16 Linux prebuilt release from AWS

### ISSUE THREAD

**Issue by sibbl** [baileyherbert/utimes#17]
I'm trying to install utimes on a Docker alpine 16 LTS setup but receive the following error message:

`#24 73.44 npm ERR! node-pre-gyp WARN Hit error response status 403 Forbidden on https://node-utimes.s3.amazonaws.com/4.1.1/Release/utimes-v4.1.1-node-v93-linux-arm64.tar.gz`

See https://github.com/sibbl/matrix-bot-media-fetcher/runs/5527172819?check_suite_focus=true

**Comment by baileyherbert** [comment]
Looks like there is an `arm` build but not `arm64`. I wasn't actually aware that the latter existed, and node-pre-gyp doesn't list `arm64` as a valid option, though it does appear to work.... 😓 

Let me toy with this and see what I can do, it will take perhaps a few hours. In the meantime you could configure npm with a valid python 3 binary and it will build the binary itself.

**Comment by sibbl** [comment]
Thanks for the very quick response! Haven't thought of the platform but this definitely makes sense...

It's not that urgent as I don't require arm64 but want to support e.g. RPi devices in the future. So I'll probably just disable the target in my project in the meantime.

So no hurry while toying around and thanks in advance for a fix!

**Comment by baileyherbert** [comment]
Hi @sibbl, apologies for the delay here. I took the opportunity to make some much-needed upgrades to my workflows! 😊 

I'm happy to announce that there are now `armv7` and `arm64` prebuilt binaries available for linux. I've just released version `5.0.0` which mainly improves error messages, but have also made the new binaries available for `4.1.1`, so your failed job should succeed with a re-run.

Let me know if you have any issues with it & cheers!
